Olori-Ebi/credit2
# Ft resources server 169157174

#### What does this PR do?
Get a single loan/Get loan by ID
 
#### Description of Task to be completed?
When you enter your route with your ID on postman, it gets you the loan with the ID entered

#### How should this be manually tested?
- Run npm install 
- Run npm start
- Run postman/install if not already installed
- Make a GET request on postman and slot in your route et ID and note that ID must be a number e.g http://localhost:3000/api/v1/loans/2

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#[169157174](https://www.pivotaltracker.com/story/show/169157174)
#### Screenshots (if appropriate)
#### Questions:
